package/common_init/overlay/etc/init.d/S20flags

```diff
@@ -0,0 +1,13 @@
+#!bin/sh
+#
+# initialize flags
+#
+
+log_tag=flag_init
```

Comment:
FYI, this log_tag variable isn't needed unless you are using the `logging.sh` support script